### ISSUE TITLE
4 cores but 8 logical processors

### ISSUE THREAD

**Issue by Bosse62** [openhardwaremonitor/openhardwaremonitor#1512]
Hi

Is there anything to do about this:

![image](https://user-images.githubusercontent.com/104043099/164127193-2f060791-55c5-49cb-8e85-e53a1cc54875.png)

![image](https://user-images.githubusercontent.com/104043099/164127261-d9b29313-8e27-4f27-8828-d68b9f012b6c.png)

Best regards
Bosse62
bosse.eilertsen@protonmail.com

**Comment by DarthDestroyer** [comment]
Yeah, its intended -> look at https://github.com/openhardwaremonitor/openhardwaremonitor/issues/1065#issuecomment-354523923
